app/services/food_service.rb

```diff
@@ -0,0 +1,16 @@
+class FoodService
+  def self.food_search(food)
+    response = conn.get("search?api_key=#{ENV['X-Api-Key']}&query=#{food}")
+    parse_json(response)
+  end
+
+  def self.conn
+    Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/foods/")
+  end
```

Comment:
for the connection, just connect to the root of the api, `https://api.nal.usda.gov`. Your connection is less likely to be usable for other endpoints when you have the url including part of the endpoint you're trying to hit. Then, in your `food_search`, you'd get `/fdc/v1/foods/search?api_key=#{ENV['X-Api-Key']}&query=#{food}`. I think you could add into your connection the api key param though, since all connections will require that api key. Lastly, this will definitely work how you have it, but ruby convention is to use block syntax when adding params to a path, so take a look at that syntax.